frame/staking/src/pallet/mod.rs

```diff
@@ -609,6 +609,8 @@ pub mod pallet {
 		Chilled(T::AccountId),
 		/// The stakers' rewards are getting paid. \[era_index, validator_stash\]
 		PayoutStarted(EraIndex, T::AccountId),
+		/// A validator has updated their preferences
+		ValidatorPrefsUpdated(T::AccountId, ValidatorPrefs),
```

Comment:
I would just call this `ValidatorPrefsSet` because it gets emitted even if someone didn't have any prefs to update.
```suggestion
		/// A validator has set their preferences.
		ValidatorPrefsSet(T::AccountId, ValidatorPrefs),
```